.github/dependabot.yml

```diff
@@ -32,3 +32,6 @@ updates:
     directory: "/typespec/"
     schedule:
       interval: "weekly"
+    ignore:
+      - dependency-name: "*"
+        versions: ["<1.0.0"]  
```

Comment:
Then we will start getting PRs to update the versions of dependencies (whichever dependency under /typespec/ has version >= 1.0.0).  Currently it is ignoring our constraint of, eg., "~0.46.0" (do not update minor versions). It is possible that it will adhere to the constraint once the versions get to >= 1.0.0. 
I'm proposing that we put in an ignore condition for now for us to avoid us getting multiple PRs for /typespec/ folder where we do not want to update the version.